jetbrains-core/src/software/aws/toolkits/jetbrains/core/filtering/ResourceFilterManager.kt

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.filtering
+
+import com.intellij.openapi.components.PersistentStateComponent
+import com.intellij.openapi.components.ServiceManager
+import com.intellij.openapi.components.State
+import com.intellij.openapi.components.Storage
+import com.intellij.openapi.project.Project
+
+@State(name = "filters", storages = [Storage("aws.xml")])
+class ResourceFilterManager : PersistentStateComponent<ResourceFilter> {
+    private var state = ResourceFilter(true, mapOf())
```

Comment:
if its empty, should we default to false?

Reply:
Yes, changed